BackEnd/src/main/kotlin/quest/marketstack/tradingapp/domain/mock/MockClosedTradeDataSource.kt

```diff
@@ -0,0 +1,77 @@
+package quest.marketstack.tradingapp.domain.mock
+
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import quest.marketstack.tradingapp.domain.closedtrades.ClosedTradeDataSource
+import quest.marketstack.tradingapp.model.ClosedTrade
+import quest.marketstack.tradingapp.model.TradeExec
+import java.time.LocalDate
+import java.time.LocalTime
+
+@Component
+@Profile("test")
+class MockClosedTradeDataSource : ClosedTradeDataSource {
```

Comment:
If you have any mock classes, those should be under the test directory, not main. so move this to test